docs/SUMMARY.md

```diff
@@ -54,4 +54,6 @@
 
 # Misc
 
-- [Misc](./misc/README.md)
+- [Overview](./misc/README.md)
+- [State Sync Dump](./misc/state_sync_dump.md)
+- [Spoon Mainnet State](./misc/mainnet_spoon.md)
```

Comment:
I would probably create a new section here 

```
# Guides
```

and perhaps even reuse the “A test chain with state taken from mainnet” title of the document.